presto-main/src/main/java/com/facebook/presto/execution/executor/TaskExecutor.java

```diff
@@ -214,6 +238,9 @@ public TaskExecutor(
                 guaranteedNumberOfDriversPerTask,
                 maximumNumberOfDriversPerTask,
                 taskPriorityTracking,
+                DEFAULT_INTERRUPT_RUNAWAY_SPLITS_TIMEOUT,
+                DEFAULT_INTERRUPTIBLE_SPLIT_PREDICATE,
+                DEFAULT_INTERRUPT_SPLIT_INTERVAL,
```

Comment:
same here; we could avoid all defaults through parameters

Reply:
Same thing RE making the unit test run quickly